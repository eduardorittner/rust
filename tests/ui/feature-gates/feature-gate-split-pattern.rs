pub fn main() {
    let a = &[1, 2, 3, 4, 5];
    let b = &[2, 3];
    let mut c = a.split_pattern(b); //~ ERROR use of unstable library feature 'split_pattern'
}
