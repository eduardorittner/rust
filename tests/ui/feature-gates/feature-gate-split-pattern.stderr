error[E0658]: use of unstable library feature 'split_pattern'
  --> $DIR/feature-gate-split-pattern.rs:4:19
   |
LL |     let mut c = a.split_pattern(b);
   |                   ^^^^^^^^^^^^^
   |
   = note: see issue #49036 <https://github.com/rust-lang/rust/issues/49036> for more information
   = help: add `#![feature(split_pattern)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0658`.
